### ISSUE TITLE
Endpoint `/user/read` not working when `email` is provided

### ISSUE THREAD

**Issue by lhy-hoyin** [CS3219-AY2324S1/ay2324s1-course-assessment-g32#175]
Gives "No user is using {email}` even when a valid user's email is provided.

**Comment by lhy-hoyin** [comment]
Issue is caused by underlying bug in `userRepoMySql.js` . 

In `getUserInfoByEmail`, `userInfo._userId` is undefined. Instead, you should use `userInfo.userId`.

It should look like this after updating.
```js
const getUserInfoByEmail = async (email) => {
  var _userId = Number();
  await findByEmail(email).then((userInfo) => { _userId = userInfo.userId });

  if (!_userId) throw 'No user is using ' + email;

  return getUserInfoById(_userId);
};
```

**Comment by venuslimm** [comment]
Resolved in #169
